framework/wazuh/__init__.py

```diff
@@ -64,13 +65,20 @@ def __eq__(self, other):
         return False
 
     def to_dict(self):
+
+        cti_auth_token_status = cti.get_auth_token_status()
```

Comment:
The CTI auth token status should be added as a class field and populated during the initialization instead of obtaining it during the conversion to dictionary